pages/css/architecture.md

```diff
@@ -50,46 +56,10 @@ lib/neat.scss
 _overrides.scss
 ```
 
-For the `util`, `typography`, `elements`, and `overrides` files, once they grow
-too large (300 lines or more) in size, split them into their own folder with
-sub files.
-
-```sh
-elements/_all.scss
-elements/_p.scss
-elements/_h.scss
-typography/_body.scss
-typography/_links.scss
-overrides/_breakpoints.scss
-overrides/_util.scss
-util/_center.scss
-util/_clearfix.scss
-```
-
-
-## Importing
```

Comment:
WTF is this doing in this section. Moved to "Specificity" for now.

Reply:
I think it was attempting to show how the files should be organized for predictability and findability, which seems useful, but awkward here.